packages/cli/src/workflows/workflows.controller.ee.ts

```diff
@@ -140,7 +144,7 @@ EEWorkflowController.post(
 			EEWorkflows.validateCredentialPermissionsToUser(newWorkflow, allCredentials);
 		} catch (error) {
 			throw new ResponseHelper.ResponseError(
-				'The workflow contains credentials that you do not have access to',
+				'The workflow you trying to save contains credentials not shared with you',
```

Comment:
Better wording: `The workflow you are trying to save contains credentials that are not shared with you`